### PR TITLE
Added YakChat connector files

### PR DESCRIPTION
**Pull request by aMatthewsYakChat**
---
#### Important: Our certification engineer, Srikanth, is currently on emergency leave. Please expect a delay with your PR. He will return January 31st, 2022. We apologize for any inconveniences.

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [X] I attest that the connector works and I verified by deploying and testing all the operations. 
- [X] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [X] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [X] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [X] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: certified-connectors/YakChat/Readme.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+
+## YakChat Connector
+The YakChat connector allows you to send and receive SMS and MMS using your YakChat numbers. YakChat adds SMS and MMS to Microsoft Teams so that you can connect instantly with your customers and co-workers using text messaging.
+
```
Comment (sriyen-msft) [inline certified-connectors/YakChat/Readme.md:4]: @aMatthewsYakChat Please refer the [templates provided for readme.md file]
(https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md) and add required sections.,

File: certified-connectors/YakChat/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,382 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "YakChat",
+    "description": "Connector to send and receive SMS and MMS using your YakChat numbers. YakChat adds SMS and MMS to Microsoft Teams so that you can connect instantly with your customers and co-workers using text messaging.",
+    "version": "1.0",
+	"contact": {
+		"name": "YakChat Ltd.",
```
Comment (sriyen-msft) [inline certified-connectors/YakChat/apiDefinition.swagger.json:8]: @aMatthewsYakChat Indentation - The formatting of both the swagger and apiProperties.json files is not correct. The properties are not aligned properly and there is a longer tab index between the properties. Please refer to the certified connectors in the [open source repo](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/Africa%27s%20Talking%20Airtime/apiDefinition.swagger.json) for reference.

File: certified-connectors/YakChat/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,382 @@
...
+              "title": "Inbox"
+            }
+          }
+        }
+      },
+      "post": {
+        "responses": {
+          "200": {
+            "description": "response",
+            "schema": {}
```
Comment (sriyen-msft) [inline certified-connectors/YakChat/apiDefinition.swagger.json:175]: @aMatthewsYakChat Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.

Applies the comment for all the similar empty schema definitions.

File: certified-connectors/YakChat/apiProperties.json [hunk 4]
```diff
@@ -0,0 +1,51 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "aad",
+          "clientId": "314772fb-53a1-4908-9b17-1692796bb13f",
```
Comment (sriyen-msft) [inline certified-connectors/YakChat/apiProperties.json:8]: @aMatthewsYakChat Please replace it with "[DUMMY]"

File: certified-connectors/YakChat/apiProperties.json [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+            "required": "false",
+            "hidden": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#fcbf00",
+    "capabilities": [],
+    "publisher": "YakChat Ltd.",
+	"stackOwner": "YakChat Ltd."
```
Comment (sriyen-msft) [inline certified-connectors/YakChat/apiProperties.json:49]: @aMatthewsYakChat Please remove the extra spaces

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/YakChat/Readme.md (hunk 1 above)
Comment (aMatthewsYakChat) [reply certified-connectors/YakChat/Readme.md:4]: This has been updated

File: certified-connectors/YakChat/apiDefinition.swagger.json (hunk 2 above)
Comment (aMatthewsYakChat) [reply certified-connectors/YakChat/apiDefinition.swagger.json]: This has been updated

File: certified-connectors/YakChat/apiDefinition.swagger.json (hunk 3 above)
Comment (aMatthewsYakChat) [reply certified-connectors/YakChat/apiDefinition.swagger.json]: This has been updated

File: certified-connectors/YakChat/apiProperties.json (hunk 4 above)
Comment (aMatthewsYakChat) [reply certified-connectors/YakChat/apiProperties.json]: This has been updated

File: certified-connectors/YakChat/apiProperties.json (hunk 5 above)
Comment (aMatthewsYakChat) [reply certified-connectors/YakChat/apiProperties.json]: This has been updated

**Comment by Amjed-Ayoub** [comment]
Dear partner,

We apologize for the delay you might be experiencing while we are working on a high volume of connector requests. Rest assured that we are doing everything we can to address this volume. We are taking measures like improving our processes and hiring additional team members to guarantee resolving your connector requests in a timely manner. 

Please let us know if you have any questions in the meantime.

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**